provider/examples/resources/sdc/main.tf

```diff
@@ -0,0 +1,15 @@
+module "example_vpc" {
+  source = "./vpc"
+  resource_prefix = "terraform-provider-example"
+}
+
+module "example_sdc" {
```

Comment:
I would add two examples here: one to spin up an SDC in AWS and one to spin it up in vSphere.

Reply:
Will do in another PR 🙂 , since vsphere sdc is broken at the moment